books/free-programming-books-it.md

```diff
@@ -157,6 +166,11 @@
 * [INFN di Milano](http://www.mi.infn.it/~calcolo/corso_base_html/pdf/corso_base_html.pdf) (PDF)
 
 
+### IBM RPG
+
+* [Strumenti per lo sviluppo WEB in RPG](https://www.easy400.net/easy400p/main.cgi) - Giovanni B. Perotti (HTML) 
```

Comment:
It should be better use HTML instead of CGI?

```suggestion
* [Strumenti per lo sviluppo WEB in RPG](https://www.easy400.net/easy400p/main.html) - Giovanni B. Perotti (HTML) 
```

Reply:
I have transcripted the original link. CGI is an old protocol for web, but the site is in html of course.